eth/ethconfig/config.go

```diff
@@ -218,6 +218,19 @@ type Config struct {
 
 	// Quorum
 	core.QuorumChainConfig `toml:"-"`
+
+	// Quorum
+	QuorumLightServer                         bool   `toml:",omitempty"`
```

Comment:
can we move it to quorum chain config and/or qlight inner structure?

Reply:
thank you for your fix